core/src/main/scala/org/apache/spark/sql/TiContext.scala

```diff
@@ -143,9 +143,16 @@ class TiContext(val session: SparkSession) extends Serializable with Logging {
       db <- meta.getDatabase(dbName)
       table <- meta.getTables(db)
     } {
+      var sizeInBytes = Long.MaxValue
+      if (loadStatistics) {
+        statisticsManager.tableStatsFromStorage(table)
+        sizeInBytes = 64 * statisticsManager.getTableCount(table.getId)
```

Comment:
I think we can change this logic to solve overflow
```
val count = statisticManager.getTableCount(table.getId)
if (Long.MaxValue / count < 64) sizeInBytes = 64 * count
```

Reply:
if we use `sizeInBytes < 0` as condition, chances are that the overflow Long value is a positive arbitrary number